src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="images" 
+        class="card__image"
+      >
+  
+      <span class="card__text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</span>
+  
+      <p class="card__small-text">
+        Код товара: 195434
+      </p>
+  
+      <div class="star">
+        <div class="stars stars--4">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
+  
+        <p class="star__text">
+          Отзывов: 5
+        </p>
+      </div>
+  
+      <div class="card__price">
+        <p class="card__price-letters">Цена:</p>
+        <p class="card__price-numbers">69 999 грн</p>
+      </div>
+  
+      <div class="card__link" data-qa="hover">
+        <a 
+          class="card__link-text" 
+          href="#"
+        >
+        Купить
+        </a>
```

Comment:
Check correct indents
```suggestion
        <a 
          class="card__link-text" 
          href="#"
        >
          Купить
        </a>
```